ci-operator/config/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main.yaml

```diff
@@ -0,0 +1,87 @@
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tests:
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: rhtap
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.14"
+  run_if_changed: ^.github/workflows/push.yaml$|^server|^operator|^ci
```

Comment:
should we also look for changes in `^e2e`?

Reply:
Fixed in da9f87d5.